src/main/java/codeu/controller/RegisterServletTest.java

```diff
@@ -16,22 +17,30 @@
     private HttpServletRequest mockRequest;
     private PrintWriter mockPrintWriter;
     private HttpServletResponse mockResponse;
+    private RequestDispatcher mockRequestDispatcher;
 
     @Before
     public void setup() throws IOException{
         registerServlet = new RegisterServlet();
+
         mockRequest = Mockito.mock(HttpServletRequest.class);
-        mockPrintWriter = Mockito.mock(PrintWriter.class);
         mockResponse = Mockito.mock(HttpServletResponse.class);
+        mockRequestDispatcher= Mockito.mock(RequestDispatcher.class);
+
+        Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/register.jsp"))
+                .thenReturn(mockRequestDispatcher);
 
-        Mockito.when(mockResponse.getWriter()).thenReturn(mockPrintWriter);
+        //mockPrintWriter = Mockito.mock(PrintWriter.class);
+        //Mockito.when(mockResponse.getWriter()).thenReturn(mockPrintWriter);
     }
 
     @Test
     public void testDoGet()throws IOException, ServletException{
         registerServlet.doGet(mockRequest,mockResponse);
 
-        Mockito.verify(mockPrintWriter).println("<h1>RegisterServlet GET request.</h1>");
+        Mockito.verify(mockRequestDispatcher).forward(mockRequest,mockResponse);
```

Comment:
Add a comment that this is validating that request dispatching to the jsp is happening.

Keep in mind that none of the JSP code is actually executing, because you're doing this in a mock dispatcher.  TBH, I actually have no idea how to test the JSP itself. That's a good area to track down if you're looking for stuff to do.